src/darwin/Framework/CHIPTests/MTROTAProviderTests.m

```diff
@@ -32,28 +32,35 @@
 #define ENABLE_OTA_TESTS 1
 #endif
 
-// TODO: Disable test005_DoBDXTransferAllowUpdateRequest until PR #26040 is merged.
+// TODO: Disable test006_DoBDXTransferAllowUpdateRequest,
+// test007_DoBDXTransferWithTwoOTARequesters and
+// test008_DoBDXTransferIncrementalOtaUpdate until PR #26040 is merged.
 // Currently the poll interval causes delays in the BDX transfer and
 // results in the test taking a long time.
-#ifdef ENABLE_TEST_005
-#undef ENABLE_TEST_005
+#ifdef ENABLE_TESTS
+#undef ENABLE_TESTS
 #endif
 
 static const uint16_t kPairingTimeoutInSeconds = 10;
 static const uint16_t kTimeoutInSeconds = 3;
 static const uint16_t kTimeoutWithUpdateInSeconds = 60;
 static const uint64_t kDeviceId1 = 0x12341234;
 static const uint64_t kDeviceId2 = 0x12341235;
+static const uint64_t kDeviceId3 = 0x12341236;
 // NOTE: These onboarding payloads are for the chip-ota-requestor-app, not chip-all-clusters-app
 static NSString * kOnboardingPayload1 = @"MT:-24J0SO527K10648G00"; // Discriminator: 1111
 static NSString * kOnboardingPayload2 = @"MT:-24J0AFN00L10648G00"; // Discriminator: 1112
+static NSString * kOnboardingPayload3 = @"MT:-24J0CEK01L10648G00"; // Discriminator: 1113
```

Comment:
Where did you get this QR code?  This one sets the discovery bitmask to BLE-only, whereas we want to set it to DNS-SD-only.

Reply:
i ran this in command line and the app prints the code. Used that here.

Reply:
> i ran the app from command line 

Well, the app was then compiled with the wrong flags for getting a QR code from it.... ;)

Please fix this code to have the right discovery bitmask.

Reply:
most likely. its surprising though because every run i have built the app i have set the ble flag as false. just ran the same command every time. But definitely something was amiss here :). Fixed